src/components/day-events/EventList.tsx

```diff
@@ -1,22 +1,24 @@
 import React, {useContext} from "react";
 import './day-event.css';
-import {MedicalCalendarContext} from "../../context/calendar-context/MedicalCalendarContext";
+import {MedicalCalendarContext} from "src/context/calendar-context/MedicalCalendarContext";
+import {CalendarContextType, CalendarEvent} from "src/models/Models";
 
-const EventList = ({events, watch}) => {
+const EventList = ({eventList, watch}:{eventList:CalendarEvent[], watch:string}) => {
```

Comment:
Aqui pode definir watch como um type union para manter a consistência do código

```suggestion
type EventListProps = {
  eventList:CalendarEvent[]
  watch: 'manha' | 'tarde' | 'noite'
}

const EventList = ({eventList, watch}: EventListProps) => {
```